### PR TITLE
Return wonky expression coersion-ishness to the way it was before

### PR DESCRIPTION
**Pull request by geoffjentry**

### CODE REVIEW

**Comment by geoffjentry** [comment]
@cjllanwarne @mcovarr 

This is the change we discussed the other night

File: scala/wdl4s/build.sbt [hunk 1]
```diff
@@ -18,7 +18,7 @@ val DowngradedSprayV = "1.3.1"
 libraryDependencies ++= Seq(
   "com.typesafe.scala-logging" %% "scala-logging" % "3.1.0",
   "io.spray" %% "spray-json" % DowngradedSprayV,
-  "org.scalaz" %% "scalaz-core" % "7.2.0",
+  "org.scalaz" %% "scalaz-core" % "7.1.3",
```
Comment (mcovarr) [inline scala/wdl4s/build.sbt]: hrm?

Comment (geoffjentry) [inline scala/wdl4s/build.sbt]: I told you about this the other day. There's a lot of changes between the 2 versions and having it out of synch with cromwell wasn't pleasant. Also I'm not upgrading cromwell for this effort.

Comment (mcovarr) [inline scala/wdl4s/build.sbt]: Cool, any effort in this arena is better spent moving to :smirk_cat: anyway

Comment (geoffjentry) [inline scala/wdl4s/build.sbt]: You win for making me laugh w/ that emoji.

File: scala/wdl4s/src/main/scala/wdl4s/RuntimeAttributes.scala [hunk 2]
```diff
@@ -11,15 +11,15 @@ import scala.language.postfixOps
...
 
 object RuntimeAttributes {
   def apply(ast: Ast): RuntimeAttributes = {
     val asts = ast.findAsts(AstNodeName.Runtime)
     if (asts.size > 1) throw new UnsupportedOperationException("Only one runtime block may be defined per task")
     val astList = asts.headOption map { _.getAttribute("map").asInstanceOf[AstList] }
-    val attrMap = astList map processRuntimeAttributes getOrElse Map.empty[String, String]
-    attrMap.get("memory") foreach { validateMemory }
+    val attrMap = astList map processRuntimeAttributes getOrElse Map.empty[String, Seq[String]]
+    attrMap.get("memory") flatMap { _.headOption } foreach { validateMemory }
```
Comment (mcovarr) [inline scala/wdl4s/src/main/scala/wdl4s/RuntimeAttributes.scala]: unnecessary curlies around `validateMemory`

**Comment by mcovarr** [comment]
:+1:

File: scala/wdl4s/src/main/scala/wdl4s/RuntimeAttributes.scala [hunk 3]
```diff
@@ -11,15 +11,15 @@ import scala.language.postfixOps
...
 
 object RuntimeAttributes {
   def apply(ast: Ast): RuntimeAttributes = {
     val asts = ast.findAsts(AstNodeName.Runtime)
     if (asts.size > 1) throw new UnsupportedOperationException("Only one runtime block may be defined per task")
     val astList = asts.headOption map { _.getAttribute("map").asInstanceOf[AstList] }
-    val attrMap = astList map processRuntimeAttributes getOrElse Map.empty[String, String]
-    attrMap.get("memory") foreach { validateMemory }
+    val attrMap = astList map processRuntimeAttributes getOrElse Map.empty[String, Seq[String]]
+    attrMap.get("memory") flatMap { _.headOption } foreach validateMemory
```
Comment (cjllanwarne) [inline scala/wdl4s/src/main/scala/wdl4s/RuntimeAttributes.scala]: I guess this is probably just "what is was before" - but any idea why memory is singled out here?

Comment (geoffjentry) [inline scala/wdl4s/src/main/scala/wdl4s/RuntimeAttributes.scala]: memory happens to be the only runtime attribute the wdl spec is opinionated about

**Comment by cjllanwarne** [comment]
:+1: with maybe a new ticket to validate other known options too?

**Comment by geoffjentry** [comment]
Going directly to the new repo

**Pull request closed without merging**